automation/settings.ps1

```diff
@@ -0,0 +1,192 @@
+# Settings for the ModpackUploader
+# For details/help see: https://github.com/NillerMedDild/ModpackUploader
+
+# The path to the main modpack folder. (the same folder as the mods folder is in)
+# Running the modpack uploader from the main modpack folder: ("$PSScriptRoot" | Resolve-Path)
+# Running the modpacker uploader from a subfolder: ("$PSScriptRoot/.." | Resolve-Path)
+$INSTANCE_ROOT = ("$PSScriptRoot/.." | Resolve-Path)
+
+# =====================================================================//
+#  CURSEFORGE ACCOUNT SETTINGS
+# =====================================================================//
+
+$CURSEFORGE_USER = "itsDinkd"
+
+# For details see: https://www.curseforge.com/account/api-tokens
+# Defined in secrets.ps1 
+# $CURSEFORGE_TOKEN = 
+
+# ProjectID can be found on the modpack's Curseforge Projects page, under "About This Project"
+$CURSEFORGE_PROJECT_ID = 507137
+
+# =====================================================================//
+#  MAIN MODPACK SETTINGS
+# =====================================================================//
+
+# This is the modpack name as seen in it's CurseForge url: https://www.curseforge.com/minecraft/modpacks/[enigmatica6]
+$MODPACK_NAME = "another-quality-modpack-2"
+
+# Name of the Modpack in the ZIP File
+$CLIENT_NAME = "AQM2"
+
+# Version Of The Modpack
+$MODPACK_VERSION = "1.6"
```

Comment:
This is the next version to be released and uploaded, it needs to be changed before executing the script